nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/autodiff/samediff/NameScopeTests.java

```diff
@@ -85,7 +85,7 @@ public void testOpFieldsAndNames(){
         assertEquals("s1/addxy", addWithName.name());
 
         assertTrue(merge.name(), merge.name().startsWith("s1/s2/"));
-        assertEquals("s1/s2/mmax", mergeWithName.name());
+        assertEquals("s1/s2/mergemax_1", mergeWithName.name());
```

Comment:
don't change the condition here, it not passing without a change IS a problem (due to previous comment)